addons/website/tests/test_disable_unused_snippets_assets.py

```diff
@@ -16,24 +16,37 @@ def setUp(self):
         self.homepage = self.View.create({
             'name': 'Home',
             'type': 'qweb',
-            'arch_db': HOMEPAGE_WITH_OUTDATED_S_WEBSITE_FORM,
+            'arch_db': HOMEPAGE_OUTDATED,
             'key': 'website.homepage',
         })
         self.mega_menu = self.WebsiteMenu.create({
             'name': 'Image Gallery V001',
-            'mega_menu_content': MEGA_MENU_CONTENT_IMAGE_GALLERY_V001,
+            'mega_menu_content': MEGA_MENU_UP_TO_DATE,
         })
 
         self.initial_active_snippets_assets = self._get_active_snippets_assets()
 
-    def test_homepage_with_outdated_s_website_form(self):
+    def test_homepage_outdated_and_mega_menu_up_to_date(self):
```

Comment:
Even in test files, I am not a fan of renaming everything (methods, global variables, etc) in stable. But maybe it is better in this case, not sure... @rdeodoo what do you think?

Reply:
I don't like it either, mainly because this is the best way to break an existing test, as the change might alter the test / the way it was testing.
In this case, it seems fine.
About the stable policy about someone that might be using those methods/records, I think this is ok, worst case we won't break enduser flow, just testing suite.
Keep it that way IMHO, especially since it is already written.